Jellyfin Player/Extensions/StringExtension.swift

```diff
@@ -0,0 +1,15 @@
+//
```

Comment:
All files need to start with the MPL license header

Reply:
```suggestion
//
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
```